spec/api-web-contents-spec.ts

```diff
@@ -2511,18 +2511,18 @@ describe('webContents module', () => {
     it('emits when moveTo is called', async () => {
       const w = new BrowserWindow({ show: false });
       w.loadURL('about:blank');
-      w.webContents.executeJavaScript('window.moveTo(100, 100)', true);
+      w.webContents.executeJavaScript('window.moveTo(50, 50)', true);
```

Comment:
This isn't blocking, but curious why 50/50 works and 100/100 doesn't, is it a CircleCI VM issue?

Reply:
@VerteDinde yes.  I'm not sure if its a window size constraint or what but it resolves the issue - and the test wasn't necessarily concerned about where the contents were moved to, just that content-bounds-updated fires